hedyorg/hedy
# [CHORE, TESTING] pretty print exception output for better readability

Fixes #3582 and will also be helpful if we ever present this to translators.

**How to test**
Deliberately destroy a code snippet in one of the snippet folders (f.e. remove a closing bracket, or misspel a keyword) and verify you get better output that `False is not True`
